helm/superset/templates/deployment-beat.yaml

```diff
@@ -72,6 +72,16 @@ spec:
         {{- end }}
       {{- if .Values.supersetCeleryBeat.initContainers }}
       initContainers: {{-  tpl (toYaml .Values.supersetCeleryBeat.initContainers) . | nindent 6 }}
+      {{- if or (.Values.extraEnvRaw) (.Values.extraEnv) }}
+        env:
```

Comment:
Will this work as expected? `initContainers` will render a list of containers, each with their own `env` block. With this new addition, you will end up with a dangling `env` after the list of containers